<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary vector sequence generator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Russian" country="Ukraine"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_n">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_in_n">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_p">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_in_p">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_funcs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Пошук</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_parametrs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>400</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметри пошуку функцій:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_possible_funcs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Логічні операції:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_and">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>AND (*)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_or">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>OR (+)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_not">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NOT (!)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_xor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR (⊕)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_functions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>400</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>Функції:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_funcs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sequence">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Згенерувати послідовність</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sequence_in_file">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>860</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Згенерувати послідовність у файл</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_vectors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>1500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
